modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -63,22 +61,21 @@
 			<param index="0" name="what" type="Variant" />
 			<param index="1" name="type" type="int" />
 			<description>
-				Converts from a type to another in the best way possible. The [code]type[/code] parameter uses the [enum Variant.Type] values.
+				Converts [param what] to [param type] in the best way possible. The [param type] uses the [enum Variant.Type] values.
 				[codeblock]
 				a = Vector2(1, 0)
-				# Prints 1
-				print(a.length())
+				print(a.length()) # Prints 1, because it returns the length of Vector2.
+
 				a = convert(a, TYPE_STRING)
-				# Prints 6 as "(1, 0)" is 6 characters
-				print(a.length())
+				print(a.length()) # Prints 6, because "(1, 0)" is 6 characters long.
```

Comment:
One more thing: this conversion (`Vector2`->`String`) doesn't work in `4.0` (a little related: #65919, #66116) so this example is incorrect and thus should be removed/changed. Possible replacement:
```gdscript
var a = Vector2(1.5, 0)
prints(a, a is Vector2, a is Vector2i) # Prints "(1.5, 0) true false".
a = convert(a, TYPE_VECTOR2I)
prints(a, a is Vector2, a is Vector2i) # Prints "(1, 0) false true".
```
Feel free to use a different example if this one is not clear or something like that.

Reply:
Using the "is" keyword to explain it is perfect, how did I not think of that? Addressed with a similar example.

Reply:
Good example but the added empty line is not really empty and it makes the checks fail.

BTW don't you have [pre-commit hooks](https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#pre-commit-hook) installed ([direct link](https://github.com/godotengine/godot/tree/master/misc/hooks))? They wouldn't let you commit it like that (I think it works for docs too?:thinking:).

Reply:
I caved in and installed LLVM and the hooks. Unfortunately, as I suspected, it's not easy as that because despite ensuring LLVM is added to PATH, the following error occurs:

```
Running hook: pre-commit-clang-format
/usr/bin/env: ‘bash’: No such file or directory
```
I have addressed that check fail manually in the meantime, but it'd be nice to solve this someday.

Reply:
I don't think I had this problem. But I can see `bash.exe` in `git/bin` which I have added to `PATH`. So [installing git](https://git-scm.com/download/win) might solve it for you? :thinking: (don't you have it installed?)